data/projects.json

```
@@ -10,8 +10,8 @@
         "tags": "dungeon, markdown-dungeon, gatsby, reactjs"
     },
     {
-        "name": "Memoy-Card-Game",
-        "display": "Memoy Card Game",
+        "name": "Memory-Card-Game",
```

Rollback this change due to the repository name is `Memoy-Card-Game`
```suggestion
        "name": "Memoy-Card-Game",
```